### PR TITLE
add homepage layout

### PR DESCRIPTION
**Pull request by simleap**
screenshot: https://goo.gl/S2nkgX

### CODE REVIEW

File: spec/system/dashboard/home_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe "home" do
+  it "has navigation links" do
+    visit(root_path)
+
+    within("#top_nav") do
+      expect(page).to have_link(
+        I18n.translate!(:"titles.home.index"),
```
Comment (dwilkie) [inline spec/system/dashboard/home_spec.rb]: no need to translate here. Just hard code the translation. E.g. 

```rb
expect(page).to have_link("Home")...
```

File: config/routes.rb [hunk 2]
```diff
@@ -8,7 +8,10 @@
              controllers: { invitations: "dashboard/user_invitations" },
              skip: :registrations
 
-  root to: redirect("dashboard")
+  root 'home#index'
+
+  get "/about_us", to: "home#about_us"
```
Comment (dwilkie) [inline config/routes.rb]: Maybe we don't need this. Let's just have a single page for now

File: app/views/layouts/home.html.haml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    %meta{:content => "width=device-width, initial-scale=1, shrink-to-fit=no", :name => "viewport"}/
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
+    = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %title= title
+  %body.text-center
+    .container.d-flex.w-100.h-100.p-3.mx-auto.flex-column
+      %header.masthead.mb-auto
+        = render 'shared/flash_messages'
+        .inner
+          %h3.masthead-brand Somleng
```
Comment (dwilkie) [inline app/views/layouts/home.html.haml]: Use translation here `titles.application_name`

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -11,7 +11,7 @@ def flash_class(level)
   def title
     translate(
       :"titles.#{controller_name}.#{action_name}",
-      id: resource && resource.id,
+      id: defined?(resource) && resource && resource.id,
```
Comment (dwilkie) [inline app/helpers/application_helper.rb]: maybe we should pass the resource, controller_name and action_name as as parameters to this method. Then we don't need to check for `defined?(resource)`

File: app/controllers/home_controller.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class HomeController < ApplicationController
+  layout "home"
+
+  def index; end
+
+  def about_us; end
```
Comment (dwilkie) [inline app/controllers/home_controller.rb]: remove `about_us` and `contact_us` for now. Let's just have a single page `home`

**Comment by samnang** [comment]
@simleap could you look at Dave's responses.

Also, could you choose this layout https://getbootstrap.com/docs/4.0/examples/pricing/ instead of the black one?

**Pull request merged**